SPECS/python-execnet/python-execnet.spec

```diff
@@ -36,23 +43,38 @@ a minimal and fast API targetting the following uses:
 %prep
 %autosetup -n %{pkgname}-%{version}
 
-%build
-%py3_build
+find . -type f -a \( -name '*.py' -o -name 'py.*' \) \
+   -exec sed -i '1{/^#!/d}' {} \; \
+   -exec chmod u=rw,go=r {} \;
+
+%pyproject_buildrequires -t
 
+%build
+%pyproject_wheel
+make -C doc html PYTHONPATH=$(pwd)/src
+# remove hidden file
+rm doc/_build/html/.buildinfo
+ 
 %install
-%py3_install
+%pyproject_install
+%pyproject_save_files %{pkgname}
 
 %check
-pip3 install tox
-sed -i "s/pytest$/pytest==7.1.3/" tox.ini
+pip3 install tox iniconfig
+# sed -i "s/pytest$/pytest==7.1.3/" tox.ini
 LANG=en_US.UTF-8 tox -e py%{python3_version_nodots}
 
-%files -n python3-%{pkgname}
-%license LICENSE
-%doc README.rst CHANGELOG.rst
-%{python3_sitelib}/*
+%files -n python%{python3_pkgversion}-%{pkgname} -f %{pyproject_files}
+%doc README.rst
+%doc doc/_build/html
+%license %{python3_sitelib}/%{pkgname}-%{version}.dist-info/licenses/LICENSE
 
 %changelog
+* Wed Apr 24 2024 Osama Esmail <osamaesmail@microsoft.com> - 2.1.1-1
+- Auto-upgrade to 2.1.1-1
```

Comment:
you can remove the release number here, and only mention the actual source number ("Auto-upgrade to 2.1.1")